### PR TITLE
Review nits/bug fixes

### PR DESCRIPTION
**Pull request by kgang**
* [bug] fixes message_format kwarg pop in qna
* Adds typhinting to orchestrator
* quality of life change to make ChatCompletion functions properties

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,14 @@ To run the celery worker, run ```make celery```. This command will also start ra
...
 
-To run operator methods through celery, run `operator.foo.delay(kwargs).get()`. It is important that arguments are keyword arguments. This returns a ConcreteChatCompletion object. message_format has been added to this object to allow for client-side validation into a message format via `ConcreteChatCompletion.get_message`()`.
\ No newline at end of file
+To run operator methods through celery, run `operator.foo.delay(kwargs).get()`. 
+```python
+from concrete import clients, operators
+
+c = {'openai': clients.OpenAIClient(temperature=0)}
+message="How moral is world domination if you're good"
+resp: ConcreteChatCompletion = operators.Operator(c).chat.delay(message=message).get()
```
Comment (MichaelDeng03) [inline README.md:40]: Might be nice to mention resp.get_message(): Message

Comment (MichaelDeng03) [reply README.md:40]: Ah nvm, I see that you've changed concretechatcompletion

Comment (kgang) [reply README.md:40]: I added `.message` to the README cuz both are valid [](edited-with-Axolo)

**Review by MichaelDeng03 — APPROVED** [review]
lgtm

**Comment by kgang** [comment]
let me fix the lint error, not sure why those didn't show up locally

**Comment by MichaelDeng03** [comment]
sweet

**Pull request merged**